client/src/views/mobile/CellMobileView.tsx

```diff
@@ -38,97 +40,140 @@ const CellMobileView = ({
   confirmReport,
   confirmSacrifice,
   confirmReset,
+  bruteList
 }: CellMobileViewProps) => {
   const { t } = useTranslation();
   const { brute, owner } = useBrute();
+  const navigate = useNavigate();
   const theme = useTheme();
   const isXs = useMediaQuery(theme.breakpoints.only('xs'));
+  const pageRef = useRef<HTMLInputElement>(null);
+
+  useEffect(
+    () => {
+      let touchstartX = 0;
+      let touchendX = 0;
+      const checkSwipe = () => {
+        if (touchendX > touchstartX && touchendX - touchstartX > -120) {
+          if (bruteList && bruteList[bruteList.indexOf(brute!.name) - 1]) {
+            navigate(`/${bruteList[bruteList.indexOf(brute!.name) - 1]}/cell`);
+          }
+        }
+        if (touchendX < touchstartX && touchstartX - touchendX > -120) {
+          if (bruteList && bruteList[bruteList.indexOf(brute!.name) + 1]) {
+            navigate(`/${bruteList[bruteList.indexOf(brute!.name) + 1]}/cell`);
+          }
+        }
+      };
+
+      document.addEventListener('touchstart', (e) => {
```

Comment:
You need to store the reference to your handlers in a variable, in order to be able to remove them as listeners later

```ts
const touchStartHandler = () => { ... };
 document.addEventListener('touchstart', touchStartHandler );

...

document.removeEventListener('touchstart', touchStartHandler );
```